src/components/home/partners/SecondaryPartners.jsx

```diff
@@ -65,25 +67,29 @@ const LargeScreenComponent = ({
             alt={`${company} logo`}
             aria-label={`${company} logo`}
             src={partnerLogo}
-            mb={'150px'}
             width={'250px'}
           ></Box>
         </a>
       </Grid>
-      <Grid item pl={'6%'} {...partnerGridStyle} order={contentOrder}>
-        <Typography variant="body1" p={'5% 15% 3% 0'}>
+      <Grid item pl={'6%'} pr={'1%'} {...partnerGridStyle} order={contentOrder}>
+        <Typography variant="body1" p={'5% 0 3% 0'}>
           {testimonial}
         </Typography>
-        <Typography variant="caption" p={'15px 0 5% 0'}>
+        <Typography variant="caption" sx={{ fontWeight: 'bold' }}>
           {testimonialAuthor}
         </Typography>
         {testimonialTwo ? (
-          <Typography variant="body1" display={'flex'} p={'40px 15% 5% 0'}>
+          <Typography
+            variant="body1"
```

Comment:
not necessary. the `Typography` MUI component does not have a `display` attribute. This does nothing.